next.config.js

```diff
@@ -1,3 +1,6 @@
+const MillionLint = require("@million/lint");
+const millionLint = process.env.USE_MILLION_LINT ? MillionLint.next() : (config) => config;
```

Comment:
Does it matter if we only import Million Lint when the environment variable is set?

```suggestion
const millionLint = process.env.USE_MILLION_LINT ? require("@million/lint").next() : (config) => config;
```

Reply:
That should work too. I can make the adjustment. I imagine they have a guard for not running in prod, but no point in loading it if you're not using it.